performant-software/DM
# Text editor display fixes

### What does this PR do?
- avoids the destruction of editable text frames upon their containers dragging by temporarily making them uneditable while a drag is in progress
- changes how text highlights are hidden and displayed so that annotation targets show up (alone) consistently

### What issues does this address?
#96, #97

### How to test
See described bugs in #96 and #97. Visit staging (http://67.207.158.85/) and test general behavior with opening text targets from annotations, dragging containers in the grid, and toggling the text editors between editable/uneditable and highlights-shown/highlights-hidden
